<launch>
    <arg name="rviz" default="false" />

    <rosparam command="load" file="$(find robot_base)/config/robot_base.yaml" />

    <node pkg="robot_base" type="robot_base_node" name="robot_base" output="screen"/>

    <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find robot_base)/rviz_cfg/robot_base.rviz" />
    </group>

</launch>